identity.go

```diff
@@ -109,7 +110,7 @@ func BasePolicy(next http.Handler) http.Handler {
 			return
 		}
 
-		if id.Identity.Internal.OrgID == "" {
+		if id.Identity.OrgID == "" || id.Identity.Internal.OrgID == "" {
```

Comment:
Should this be `&&`?

Rationale is, if you have the old one `Internal.OrgID` we should copy it to the new location.

Reply:
But isn't this check failing if either are missing? It sounds like our rationale aligns.

Reply:
Yes, that'd support old clients.

Reply:
@jhjaggars added the fallback for old clients